moduling/render.js

```diff
@@ -6,3 +6,4 @@ const result = checkArray([1,2,3]);
 
 //log output into the console.
 console.log(result);
```

Comment:
I am guessing that you used the `var` to test that the line is being flagged by eslint. After testing, you would need to remove them. Anyone who wants to test that eslint is in perfect working condition will now be the one to enter stuff on their own.

Also, there is one more thing I'd like you to do for this `eslint` story, and that is to explore two ways we can disable certain eslint rules. For example, say we are going to use a lot of console statements in a project (even though it is bad practice, and we don't want eslint to flag it, we go to where the line is, hover over the console statement, see the `error message` eslint produces, then we disable either in all places in the file or for that particular line.
- For disabling for a particular line, just before the line, you add a comment:
// eslint-disable-next-line no-console 
(You will see that `no-console` is what popped up when you hovered over the console statement. That was why we used no-console. So for any rule, we use what popped up when we hovered on the line.)
- For disabling throughout the particular file, It is preferably done at the top, and it takes the structure of a `multi-line comment`:
/* eslint-disable no-console */
(notice the space after the comments before the `eslint...`)

- The final method for disabling rules is to go inside the `.eslintrc`, add a key `rules`, on the values side, open an object, then make a key called `"no-console":` and a value `0`. Of course, if you are doing this, then you won't need any disabling in any of the files for that particular rule.

Test all the methods listed above, and let me know if anything is unclear, or if you face any challenge in understanding my jabber.

Reply:
You can go to my repo https://github.com/oreoluwade/petproj and check my `.eslintrc` file to see an example of what I described above. Don't worry about mine being more populated than yours; all these configurations are usually added to based upon need.

Reply:
I do not understand the second point on disabling console on multi-line comment. The (notice the space after the comments before the eslint...) was not very clear

Reply:
Another thing, at the end of each line, there is an error (unexpected linebreaks to be 'LF', but fount CRLF). How do i fix that?

Reply:
Sorry fixed the linebreak  problem

Reply:
About the space, what I meant is that you should put a space between `//` and `eslint-disable`, so that you have `// eslint-disable` and not `//eslint-disable`

Reply:
Now, I'd hope that you will fill in the detail about `blockers` in the pull request description with the "linebreak" challenge you had.

Reply:
I have done that. Merge request?